src/condor_ce_config_generator

```diff
@@ -210,28 +210,8 @@ CONFIG_TEMPLATE = """\
 #  condor_ce_config_generator
 #
 
-#
-# The following customizations can be made in a SEPARATE file:
-#
-
-# To BAN a host, add a line of the form:
-#
-#   DENY_ADVERTISE_SCHEDD = $(DENY_ADVERTISE_SCHEDD), ce.example.com@users.opensciencegrid.org
-#
-# You can have multiple comma-separated hosts in the DENY listing.
-
-# To ADD a custom host, add a line of the form:
-#
-#   COLLECTOR.ALLOW_ADVERTISE_SCHEDD = $(COLLECTOR.ALLOW_ADVERTISE_SCHEDD), ce.example.com@users.opensciencegrid.org/ce.example.com
-#
-# You can have multiple comma-separated hosts in the ALLOW listing.
-
-# The generated ALLOW_ADVERTISE_SCHEDD should be of the form:
-#
-#   COLLECTOR.ALLOW_ADVERTISE_SCHEDD = foo.example.com@users.opensciencegrid.org/foo.example.com, bar.example.com@users.opensciencegrid.org/bar.example.com
-#
-# There will be one entry per enabled service found in MyOSG.  This file is updated
-# atomically; MyOSG outages will not result in partial or incorrect updates.
+# There will be one entry per active, registered CE in OSG Topology.  This file is updated
```

Comment:
Is this comment going to be inside the generated file? If so, I think the first sentence is the only one that is relevant to the admin; the rest are implementation details.

Reply:
I agree, the original author was overly verbose here :P